keycloak/themes/hmda/login/login-verify-email.ftl

```diff
@@ -0,0 +1,21 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout; section>
+    <#if section = "title">
+        ${msg("emailVerifyTitle")}
+    <#elseif section = "header">
+        ${msg("emailVerifyTitle")}
+    <#elseif section = "form">
+      <div class="usa-width-one-whole margin-bottom-1">
+        <div class="usa-alert usa-alert-${message.type}">
+          <div class="usa-alert-body">
+            <h3 style="margin-top: 0;">Verify your email to activate your account</h3>
+            <p>You have been sent an email with a verification code.</p>
+            <p>This code will expire in 10 minutes. If you would like a new code or if you haven't received the email, <strong><a style="text-transform: lowercase;" href="${url.loginAction}">${msg("doClickHere")}</a></strong> to send a new one.</p>
```

Comment:
I think this should be `60 minutes`.

From the email `This link will expire within 60 minutes.`.